src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -69,6 +72,29 @@ class Orchestrator(RESTController):
     def status(self):
         return OrchClient.instance().status()
 
+    @Endpoint(method='POST')
+    @UpdatePermission
+    @raise_if_no_orchestrator
+    @handle_orchestrator_error('osd')
+    @orchestrator_task('identify_device', ['{hostname}', '{device}'])
+    def identify_device(self, hostname, device, duration):
+        # type: (str, str, int) -> None
+        """
+        Identify a device by switching on the device light for 10 seconds.
+        :param hostname: The hostname of the device to process.
+        :param device: The device identifier to process, e.g. ``ABC1234DEF567-1R1234_ABC8DE0Q``.
+        :param duration: The duration in seconds how long the LED should flash.
+        """
+        orch = OrchClient.instance()
+        TaskManager.current_task().set_progress(0)
+        orch.blink_device_light(hostname, device, 'ident', True)
```

Comment:
This method seems missing in the PR.

Reply:
Uahhh, seems to be lost while rebasing. Fixed.